src/core/objects/object_mover.cpp

```diff
@@ -102,7 +111,30 @@ void ObjectMover::move(const MapCoordF& cursor_pos, bool move_opposite_handles,
 }
 
 
-void ObjectMover::move(qint32 dx, qint32 dy, bool move_opposite_handles)
+namespace {
+bool forms_a_corner(const MapCoord anchor_point, const MapCoord moved_handle,
+                    const MapCoord opposite_handle, const qreal quantum_size)
```

Comment:
`forms_a_corner` does not conform to our spelling of function names.

I'm not so happy with having this function between the ObjectMover member function definitions, but I wonder what is the best place/namespace to put this function to. It might be a candidate for a static function in MapCoord, or at least a free function in map_coord.h. I realized that functions in the anonymous namespace are unaccessible to unit tests, too...

Reply:
Ack. I originally thought I'll reuse the function in OCD export code for corner point generation but later my thoughts went in different direction. I'll take another at the function naming and placement.

Reply:
This is now `Util::pointsFormCorner()`.